packages/python-packages/doc-warden/README.md

```diff
@@ -179,12 +180,13 @@ known_content_issues:
   - ['sdk/template/azure-sdk-template/README.md','#1368']
   - ['sdk/template/azure-sdk-template/CHANGELOG.md','#1368']
 ```
-
 The above configuration tells `warden`...
 
 - The language within the repo is `java`
 - To ensure that a `README.md` is present at the root of the repository.
-- To omit any paths under `archive/` from the readme checks.
+- To omit **any** paths under `archive/` from the readme checks.
+- To omit paths found **directly** under `sdk/eventhub/`. 
```

Comment:
For clarity perhaps you should say "omit files" instead of paths because if I understand correctly this simply omits any files directly in that folder. I must admit this is very subtle difference.